cms/envs/common.py

```diff
@@ -994,6 +994,9 @@
 XBLOCK_SETTINGS = {
     "VideoDescriptor": {
         "licensing_enabled": FEATURES.get("LICENSING", False)
+    },
+    'VideoModule': {
+        'YOUTUBE_API_KEY': YOUTUBE_API_KEY
```

Comment:
@andy-armstrong YOUTUBE_API_KEY is required by v3.0 and we will have real keys on stage, prod and edge but I was wondering about dev-stack and sandboxes like environment, for now i've set a dummy value for key and request to v3.0 will return 400 which should't be the case. 
Any suggestion around this would be helpful ?